CS_Ed_Week/code-examples/repeat_string.py

```diff
@@ -1,7 +1,8 @@
-def Repeat_String(input_number, input_string):
+def repeat_string(input_number, input_string):
   temp_list = []
-  for x in range(input_number):
+  for _ in range(input_number):
```

Comment:
I think having an underscore as the variable name is sort of confusing. I think we should stick with alphabet characters or mixtures of alphabet, numbers, and symbols for variables names, and not single symbols, as that could be mistaken for an operator of some sort.

Reply:
this is common practice for unused "throwaway" variables, I don't think it's that hard to explain it's just a variable but I guess it's also not too necessary